docs/integrations/sources/klaviyo.md

```diff
@@ -12,12 +12,12 @@ This page contains the setup guide and reference information for the Klaviyo sou
 ### Step 1: Set up Klaviyo
 
 1. Create a [Klaviyo account](https://www.klaviyo.com)
-2. Create a [Private API key](https://help.klaviyo.com/hc/en-us/articles/115005062267-How-to-Manage-Your-Account-s-API-Keys#your-private-api-keys3). Make sure you selected all [scopes](https://help.klaviyo.com/hc/en-us/articles/7423954176283) corresponding to the streams you would like to replicate.
+2. Create a [Private API key](https://help.klaviyo.com/hc/en-us/articles/115005062267-How-to-Manage-Your-Account-s-API-Keys#your-private-api-keys3). Make sure you selected all [scopes](https://help.klaviyo.com/hc/en-us/articles/7423954176283) corresponding to the streams you would like to replicate. You can find which scope is required for a specific stream by navigating to the relevant API documentation for the streams Airbyte supports.
 
 ### Step 2: Set up the Klaviyo connector in Airbyte
 
 1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
-2. Click **Sources** and then click **+ New source**.
+2. Click **Sources** and then click **+ new source**.
```

Comment:
Locally, I have a different behavior than prod hence, I can't validate this fix.

**Locally**
![image](https://github.com/airbytehq/airbyte/assets/3360483/64478206-56d8-4cb7-82f0-aa2a134f9e11)

**Prod**
![image](https://github.com/airbytehq/airbyte/assets/3360483/0bfa29bd-6677-4da4-a47e-f073add1a587)

How does removing the capital `N` removes the bullet list?

Reply:
Me either, unfortunately. This is just my assumption based on what I've seen on other connectors which don't have such issue.

Reply:
Ok! Keep me posted on the result of this change and we'll update the checklist based on that. Thanks!